src/content/content.service.ts

```diff
@@ -57,6 +61,35 @@ export class ContentService {
     return transformContent(ContentType.glossaryItem, entries)
   }
 
+  async getAiContentPrompts() {
+    const prompts = await this.prisma.content.findMany({
+      where: {
+        OR: [
+          {
+            type: ContentType.onboardingPrompts,
+          },
+          {
+            type: ContentType.candidateContentPrompts,
+          },
+        ],
+      },
+    })
+
+    // should be one content record for each "type" of prompt
+    if (prompts.length !== 2) {
+      throw new NotFoundException('Prompt content not found')
+    }
+
+    try {
+      return {
+        ...(prompts[0].data as Prisma.JsonObject),
+        ...(prompts[1].data as Prisma.JsonObject),
+      }
+    } catch (_e) {
+      throw new InternalServerErrorException('Failed to load prompts')
+    }
```

Comment:
This is just a try wrapping around a `return`. The only thing that could possibly `throw` here would be trying to spread on `.data`, if for some reason `.data` wasn't returned from Contentful for some reason. In that case, it should be handled as a `BadGatewayException` as well in the `if` block above.